modules/src/ics02_client/raw.rs

```diff
@@ -44,3 +44,15 @@ impl From<ConnectionIds> for RawClientConnections {
         }
     }
 }
+
+impl FromIterator<ConnectionId> for ConnectionIds {
```

Comment:
Haven't  noticed this file before. Why is it called `raw.rs` and part of the `ics02_client`? Its content relates to connections. It would be nice to move this to `ics03_connection`.

Reply:
This file was added during one of the first queries, `query client connections` I think in #169.

The `RawClientConnections` connections and `ConnectionIds` are the main types in this file. It's not really clear to me at this point where it would be best to place these types. (ICS03 is better than ICS02, but still this file contains a raw/proto type, which is the weirdest part that should be solved first.) I propose we refactor this and move stuff around as necessary in https://github.com/informalsystems/ibc-rs/issues/425.

Reply:
`RawClientConnections` is defined locally, not part of any protofbuf. And it is not what the name suggests, it's just  a vector of `String` -s. I'm not sure we need it at all to be honest. If we keep it, ics03 is the best place. I will take care of cleaning it in the next PR.